packages/ui/src/components/molecules/ProductTitle/styles.scss

```diff
@@ -0,0 +1,55 @@
+[data-fs-product-title] {
+  // --------------------------------------------------------
+  // Design Tokens for Product Title
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-product-title-text-size            : var(--fs-text-size-title-product);
+  --fs-product-title-text-weight          : var(--fs-text-weight-regular);
+  --fs-product-title-line-height          : 1.12;
+  --fs-product-title-column-gap           : var(--fs-spacing-2);
+  --fs-product-title-row-gap              : var(--fs-spacing-3);
+
+  --fs-product-title-addendum-margin-top  : var(--fs-spacing-3);
+  --fs-product-title-addendum-color       : var(--fs-color-text-light);
+  --fs-product-title-addendum-size        : var(--fs-text-size-1);
+  --fs-product-title-addendum-line-height : 1.7;
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
+
+  [data-fs-product-title-header] {
+    display: flex;
+    justify-content: space-between;
+    max-width: 42ch;
+    flex-wrap: wrap;
+    column-gap: var(--fs-product-title-column-gap);
+    row-gap: var(--fs-product-title-row-gap);
+
+    // Title prop
+    & > *:first-child {
+      font-size: var(--fs-product-title-text-size);
+      font-weight: var(--fs-product-title-text-weight);
+      line-height: var(--fs-product-title-line-height);
+      letter-spacing: .01em;
```

Comment:
Don't know if we should add this, because it depends a lot of the font rendered.

Reply:
Agreed! Let's leave it to the project.